salt/orch/update-etc-hosts.sls

```diff
@@ -1,19 +1,19 @@
-{% if salt.saltutil.runner('mine.get', tgt='P@roles:kube-(master|minion) and G@bootstrap_complete:true', fun='caasp_fqdn', tgt_type='compound')|length > 0 %}
+{% if salt.saltutil.runner('mine.get', tgt='P@roles:kube-(master|minion) and G@bootstrap_complete:true and not G@update_in_progress:true', fun='caasp_fqdn', tgt_type='compound')|length > 0 %}
```

Comment:
Makes sense, though, given this is a P1, we should do this after we've got the P1 closed out!